netutils/nng/Makefile

```diff
@@ -88,7 +91,8 @@ CSRCS += $(NNG_SRCDIR)/platform/posix/posix_tcpdial.c
 CSRCS += $(NNG_SRCDIR)/platform/posix/posix_tcplisten.c
 CSRCS += $(NNG_SRCDIR)/platform/posix/posix_thread.c
 CSRCS += $(NNG_SRCDIR)/platform/posix/posix_udp.c
-CSRCS += $(NNG_SRCDIR)/platform/posix/posix_pollq_epoll.c
+#CSRCS += $(NNG_SRCDIR)/platform/posix/posix_pollq_epoll.c
```

Comment:
let's wait @CV-Bowen 's analysis?

Reply:
I agree that we need to analyse the epoll functionality for possible bugs, but at the same time NNG should work. As NNG supports different mechanisms for event notifications, I suggest we move to poll in order to make example work. AFAIK poll is more efficient in case we have small number of file descriptors. And this is the case here.

Reply:
I was thinking that too. There is quite a few things we can make configurable.

Reply:
please do not comment the lines! remove them if they are not needed